schema/equals_test.go

```diff
@@ -60,11 +60,15 @@ func TestEquals(t *testing.T) {
 	// The "copy known fields" section of these function is to break if folks
 	// add new fields without fixing the Equals function and this test.
 	funcs := []interface{}{
-		func(x Schema) bool {
+		func(x *Schema) bool {
+			if x == nil {
+				// skip nil checks as the method can not be called on nil
```

Comment:
i'm not sure if this is the right way to fix the test.
but quick.Check has no easy way of telling it to not use nil.

Reply:
:man_shrugging: